toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1717,6 +1736,15 @@ func selinuxRelabelFiles(systemConfig configuration.SystemConfig, installChroot
 		defer bindMount.Close()
 
 		err = installChroot.UnsafeRun(func() error {
+			runtime.LockOSThread()
+			defer runtime.UnlockOSThread()
+
+			err := setSELinuxExecContext("system_u:system_r:setfiles_mac_t:s0")
+			if err != nil {
+				return fmt.Errorf("failed to set SELinux exec context:\n%w", err)
+			}
+			defer resetSELinuxExecContext()
```

Comment:
Could we add a comment about what's happening here? I'm guessing it may be obvious for people familiar with SELinux, but adding something for other folks may help.

Reply:
Sorry, didn't mean to push this.

(This is hack to allow the code to run on Fedora. But it isn't applicable to Ubuntu. I need to figure out how to detect which distro we are running on in a sensible way.)

Reply:
fixed